src/m365/spe/commands/containertype/containertype-get.ts

```diff
@@ -0,0 +1,154 @@
+    const formDigestInfo: FormDigestInfo = await spo.ensureFormDigest(spoAdminUrl, logger, undefined, this.debug);
+
+    const requestOptions: CliRequestOptions = {
+      url: `${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery`,
+      headers: {
+        'X-RequestDigest': formDigestInfo.FormDigestValue
+      },
+      data: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="49" ObjectPathId="48" /><Method Name="GetSPOContainerTypeById" Id="50" ObjectPathId="48"><Parameters><Parameter Type="Guid">{${containerTypeId}}</Parameter><Parameter Type="Enum">1</Parameter></Parameters></Method></Actions><ObjectPaths><Constructor Id="48" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}" /></ObjectPaths></Request>`
+    };
+
+    const res: string = await request.post(requestOptions);
+    const json: ClientSvcResponse = JSON.parse(res);
+    const response: ClientSvcResponseContents = json[0];
+
+    if (response.ErrorInfo) {
+      throw response.ErrorInfo.ErrorMessage;
+    }
+
+    const containerTypes: ContainerTypeProperties[] = json[json.length - 1];
+    return containerTypes;
+  }
+
+  private async getContainerTypeId(options: Options, spoAdminUrl: string, logger: Logger): Promise<string> {
+    if (options.id) {
+      return options.id;
+    }
+
+    return this.getContainerTypeIdByName(options.name!, spoAdminUrl, logger);
+  }
+
+  private async getContainerTypeIdByName(name: string, spoAdminUrl: string, logger: Logger): Promise<string> {
+    const formDigestInfo: FormDigestInfo = await spo.ensureFormDigest(spoAdminUrl, logger, undefined, this.debug);
+
+    const requestOptions: CliRequestOptions = {
+      url: `${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery`,
+      headers: {
+        'X-RequestDigest': formDigestInfo.FormDigestValue
+      },
+      data: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="46" ObjectPathId="45" /><Method Name="GetSPOContainerTypes" Id="47" ObjectPathId="45"><Parameters><Parameter Type="Enum">1</Parameter></Parameters></Method></Actions><ObjectPaths><Constructor Id="45" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}" /></ObjectPaths></Request>`
+    };
+
+    const res: string = await request.post(requestOptions);
+    const json: ClientSvcResponse = JSON.parse(res);
+    const containerTypes: ContainerTypeProperties[] = json[json.length - 1];
+
+    if (!containerTypes.find(c => c.DisplayName === name)) {
+      throw new Error(`Container type with name '${name}' not found`);
+    }
+    const match = containerTypes.find(c => c.DisplayName === name)!.ContainerTypeId.match(/\/Guid\(([^)]+)\)\//);
+    return match![1];
+  }
+}
```

Comment:
@reshmee011 I just saw an awesome improvement when reviewing other PR's. I noticed in @nanddeepn in his PR he refactored to get all container types to an util 
https://github.com/pnp/cli-microsoft365/pull/6412/files?diff=unified&w=0#diff-40e186d5ac89ee5a7afb93d0d6acb72b5614f059fef9e584dc61497b90d72e0f

we could reuse the same util here instead of having the same logic in command.
No action is required from your side I just wanted to let you know that I will put merging this 'on hold' until I merge #6412 and then I will refactor this PR to reuse @nanddeepn util and approach. 

Thank you both for awesome contributions and great team work 👏👏👏👏